klippy/webhooks.py

```diff
@@ -320,6 +331,24 @@ def get_status(self, eventtime):
         state_message, state = self.printer.get_state_message()
         return {'state': state, 'state_message': state_message}
 
+    def call_remote_method(self, method, **kwargs):
+        if method not in self._remote_methods:
+            raise homing.CommandError(
```

Comment:
Very minor, but it's preferable to `raise self.printer.command_error()` than to use the direct `homing` reference.

In any case, the code as a whole looks okay to me.

Thanks,
-Kevin

Reply:
I fixed this and squashed everything into 3 commits.  Thanks!